### PR TITLE
Project tooling setup (feature/MP-21)

### PR DESCRIPTION
**Pull request by dominikmlynarczyk**
Check issue for details.

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
-import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 
 ReactDOM.render(
   <React.StrictMode>
     <App />
   </React.StrictMode>,
-  document.getElementById('root')
+  document.getElementById('root'),
```
Comment (mfurmanit) [inline src/index.tsx:10]: Unnecessary comma.

**Review by mfurmanit — APPROVED** [review]
Looks great. :blush:

File: src/index.tsx (hunk 1 above)
Comment (dominikmlynarczyk) [reply src/index.tsx:10]: It is not unnecessary, look at `.prettierrc`, you'll find this:
`"trailingComma": "all",`
It gives you better diffing in Git, when you change object properties etc.
Thx for approve.

**Comment by kacpergadula** [comment]
I agree with Maciej 😊

**Review by kacpergadula — APPROVED** [review]

**Pull request merged**